product_attribute_description/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "Product Attribute Description",
+    "version": "15.0.1.0.0",
+    "category": "Website",
+    "author": "ERP Harbor Consulting Services,"
+    "Nitrokey GmbH,"
+    "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/product-attribute",
+    "summary": """
+    Product Attribute Description
+    This module Product Attribute Description is a part of product and used to show
+    attribute description of the product.
+     """,
+    "depends": ["website_product_attribute_description", "product"],
```

Comment:
You have recursive dependency here:

```suggestion
    "depends": ["product"],
```

Reply:
Okay, I'll fix the recursive dependency.